GPU/Vulkan/FramebufferVulkan.cpp

```diff
@@ -583,36 +670,31 @@ void FramebufferManagerVulkan::ResizeFramebufFBO(VirtualFramebuffer *vfb, u16 w,
 	}
 
 	textureCache_->ForgetLastTexture();
-
 	if (!useBufferedRendering_) {
 		if (vfb->fbo_vk) {
 			delete vfb->fbo_vk;
-			vfb->fbo_vk = 0;
+			vfb->fbo_vk = nullptr;
 		}
 		return;
 	}
 
 	vfb->fbo_vk = new VulkanFBO();
-	// bo_create(vfb->renderWidth, vfb->renderHeight, 1, true, (FBOColorDepth)vfb->colorDepth);
+	vfb->fbo_vk->Create(vulkan_, rpLoadColorLoadDepth_, vfb->renderWidth, vfb->renderHeight, framebufFormat);
 	if (old.fbo_vk) {
 		INFO_LOG(SCEGE, "Resizing FBO for %08x : %i x %i x %i", vfb->fb_address, w, h, vfb->format);
 		if (vfb->fbo_vk) {
 			/// fbo_bind_as_render_target(vfb->fbo_vk);
-			ClearBuffer();
-			if (!g_Config.bDisableSlowFramebufEffects) {
-				BlitFramebuffer(vfb, 0, 0, &old, 0, 0, std::min(vfb->bufferWidth, vfb->width), std::min(vfb->height, vfb->bufferHeight), 0);
-			}
+			//ClearBuffer();
+			//if (!g_Config.bDisableSlowFramebufEffects) {
+			//	BlitFramebuffer(vfb, 0, 0, &old, 0, 0, std::min(vfb->bufferWidth, vfb->width), std::min(vfb->height, vfb->bufferHeight), 0);
+			//}
 		}
+		old.fbo_vk->Destroy(vulkan_);
 		delete old.fbo_vk;
 		if (vfb->fbo_vk) {
 			// fbo_bind_as_render_target(vfb->fbo_vk);
```

Comment:
I guess we probably don't need to do this?  It'll start the render pass later.

-[Unknown]